app/design/frontend/kisilenko/kalexey_theme/web/css/source/styles-l.css

```diff
@@ -0,0 +1,3 @@
+.promo-block{
```

Comment:
These styles seems doesn't work, because they are not in the original Magento scss files which could be merged and there is no inclusion of this file for any page.

Reply:
In general, everything is fixed in 4 homework :)